### DISCUSSION TITLE
Installing from source with a compiled version of OpenCV

### DISCUSSION THREAD

**Discussion by KostadinovShalon** [open-edge-platform/anomalib#1039] [Feature Requests]
Hello, I have been recently working with a compiled version of OpenCV (with GPU capabilities) and anomalib. I realized that if I install locally via cloning the repository and performing `pip install -e .` then OpenCV is installed again (removing my compiled installation). 

I suggest checking if OpenCV is installed using the setup.py script, [as it is done here in mmcv](https://github.com/open-mmlab/mmcv/blob/89a264527e3dc9c5eebed6195faa709d446c7a9c/setup.py#L122). At the moment, I had to remove the requirement from requirements/base.txt, but I think that checking for cv2 in setup.py is cleaner.

**Comment by samet-akcay** [comment]
Nice one! Thanks for the feedback @KostadinovShalon!

**Reply by samet-akcay** [reply]
Would you like to make the contribution, or prefer us to do it ?

**Reply by KostadinovShalon** [reply]
Hi Samet, I have created a pull request adding this contribution. Cheers!

**Reply by samet-akcay** [reply]
Thanks Brian, much appreciated!
